src/main/resources/avro/genotypephenotypemethods.avdl

```diff
@@ -0,0 +1,115 @@
+
+@namespace("org.ga4gh")
+/**
+This protocol defines genotype to phenotype relationships
+query system.
+
+The goal here is to allow users to query systems using one
+or more of Genotype, Phenotype, Environment, and Evidence.
```

Comment:
Maybe this comment should be moved/echoed down where the query object is defined? So it shows up in the avrodoc for the query object?